panedr/panedr.py

```diff
@@ -395,21 +395,21 @@ def edr_strings(data, file_version, n):
 
 def is_frame_magic(data):
     """Unpacks an int and checks whether it matches the EDR frame magic number
-    
+
     Does not roll the reading position back.
     """
     magic = data.unpack_int()
     return magic == -7777777
 
 
-def edr_to_df(path, verbose=False):
+def read_edr(path, verbose_set=False):
```

Comment:
Why the change from `verbose`?

Reply:
I felt weird about `verbose=verbose`, but yeah, I guess there is not really a reason not to put this :sweat_smile: